packages/react/src/experimental/useActiveWalletController.ts

```diff
@@ -0,0 +1,14 @@
+import { ActiveWallet } from '@lens-protocol/domain/use-cases/wallets';
+
+import { useSharedDependencies } from '../shared';
+
+/**
+ * Returns the internal active wallet interactor.
+ *
+ * @internal
+ */
+export function useActiveWalletController(): ActiveWallet {
+  const { activeWallet } = useSharedDependencies();
```

Comment:
Maybe not for now, but wondering if we should just export `useSharedDependencies` (as `@internal`).

Reply:
yeah, maybe not a bad idea